ravenframework/Optimizers/Optimizer.py

```diff
@@ -237,6 +240,7 @@ def _localWhatDoINeed(self):
     needDict['Distributions'] = [(None,'all')]
     needDict['Functions'    ] = [(None,'all')]
     needDict['DataObjects'  ] = [(None,'all')]
+    needDict['Files'        ] = [(None,'all')]
```

Comment:
It seems we can do a loop over self.optAssemblerList here.

Reply:
good point